src/lua-function-list.json

```diff
@@ -296,6 +297,7 @@
     "permScript": "permScript(name, parent, lua code)",
     "permSubstringTrigger": "permSubstringTrigger( name, parent, pattern table, lua code )",
     "permTimer": "permTimer(name, parent, seconds, lua code)",
+    "playMedia": "playMedia(settings table)",
```

Comment:
Would be nice to get documentation on all of the keys that playMedia accepts - if you're not coming from an MCMP background, there's not enough context described right now.

Just as an example I set `type = "music", continue = "true"` expecting my sound to loop, but it didn't.

Reply:
Doing a better job at that in this PR and similar text will be in the wiki soon!